package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+
+.. _`10.1002/prot.340090204`: http://dx.doi.org/10.1002/prot.340090204
+
+
+How to perform WaterBridgeAnalysis
+-----------------------------------
+
+All water bridges between arginine and aspartic acid can be analysed with ::
+
+  import MDAnalysis
+  import MDAnalysis.analysis.hbonds
+
+  u = MDAnalysis.Universe('topology', 'trajectory')
+  w = MDAnalysis.analysis.hbonds.WaterBridgeAnalysis(u, 'resname ARG', 'resname ASP', distance=3.0, angle=120.0)
+  w.run()
+
+.. Note::
+
+   Due to the way :class:`WaterBridgeAnalysis` is implemented, it is
+   more efficient to have the second selection (`selection2`) be the
+   *larger* group, e.g. the whole protein when looking at ligand-protein
+   water bridge interactions.
+
+
+The results are stored as the attribute
+:attr:`WaterBridgeAnalysis.timeseries`; see :ref:`Analysis Output` for the
+format and further options.
+
+For further data analysis it is convenient to process the
+:attr:`~WaterBridgeAnalysis.timeseries` data into a normalized table with the
+:meth:`~WaterBridgeAnalysis.generate_table` method, which creates a new data
+structure :attr:`WaterBridgeAnalysis.table` that contains one row for each
+observation of a hydrogen bond::
+
+  w.generate_table()
+
+This table can then be easily turned into, e.g., a `pandas.DataFrame`_, and
+further analyzed::
+
+  import pandas as pd
+  df = pd.DataFrame.from_records(w.table)
+
+For example, plotting a histogram of the hydrogen bond angles and lengths is as
+simple as ::
+
+  df.hist(column=["angle", "distance"])
+
+.. _pandas.DataFrame: http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.html
+
+
+.. TODO: notes on selection updating
+
+
+Classes
+-------
+
+.. autoclass:: WaterBridgeAnalysis
+   :members:
+
+   .. attribute:: timesteps
+
+      List of the times of each timestep. This can be used together with
+      :attr:`~WaterBridgeAnalysis.timeseries` to find the specific time point
+      of a water bridge existence, or see :attr:`~WaterBridgeAnalysis.table`.
+
+   .. attribute:: table
+
+      A normalised table of the data in
+      :attr:`WaterBridgeAnalysis.timeseries`, generated by
+      :meth:`WaterBridgeAnalysis.generate_table`. It is a
+      :class:`numpy.recarray` with the following columns:
+
+          0. "time"
+          1. "donor_index"
+          2. "acceptor_index"
+          3. "donor_resnm"
+          4. "donor_resid"
+          5. "donor_atom"
+          6. "acceptor_resnm"
+          7. "acceptor_resid"
+          8. "acceptor_atom"
+          9. "distance"
+          10. "angle"
+
+      It takes up more space than :attr:`~WaterBridgeAnalysis.timeseries` but
+      it is easier to analyze and to import into databases or dataframes.
+
+
+      .. rubric:: Example
+
+      For example, to create a `pandas.DataFrame`_ from ``h.table``::
+
+         import pandas as pd
+         df = pd.DataFrame.from_records(w.table)
+
+
+      .. versionchanged:: 0.17.0
```

Comment:
remove this note, does not apply to a new class

Reply:
@orbeckst Thank you for the suggestion. Notes removed.